weblate/templates/mail/billing_expired.html

```diff
@@ -52,7 +52,7 @@
           {% if billing.plan.price %}
             <a href="{% url 'create-billing' %}?billing={{ billing.pk }}&amp;upgrade=1" class="button">{% trans "New payment" %}</a>
           {% else %}
-            <a href="{{ billing.get_absolute_url }}" class="button">{% trans "Request approval for Libre hosting" %}</a>
+            <a href="{{ billing.get_absolute_url }}" class="button">{% trans "Request approval for libre hosting" %}</a>
```

Comment:
No need to make the initials of “libre ” in lower case.
“Libre hosting” refers to the gratis **Libre plan** and they have already been consistent: https://hosted.weblate.org/languages/en/weblate/search/?q=Libre+hosting&sort_by=-priority%2Cposition&checksum=

Reply:
Doesn't that just undermine the idea of "libre" as a word…?
If it was called "libre plan" everywhere, it wouldn't get confused with "Hosted Weblate" maybe?
Similarly, "Billing plan" could be just "plan"(?)

Reply:
I do not think this is a good place to discuss the naming of the service provided by weblate.org.

Reply:
This is a great place to discuss naming of services. It is how it came to be called "libre" in the first place.
The big "L" is an afterthought, I imagine to avoid "libre hosting" to sound different from say "proprietary hosting".
Ultimate consistency is understandable language, everywhere.

Reply:
Pull request changing a bunch of other things is definitely not a place to discuss naming of plans.

```suggestion
            <a href="{{ billing.get_absolute_url }}" class="button">{% trans "Request approval for Libre hosting" %}</a>
```